src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationOptions.cs

```diff
@@ -48,6 +48,19 @@ public void AddScheme(string name, Action<AuthenticationSchemeBuilder> configure
         }
 
         /// <summary>
+        /// Adds an <see cref="AuthenticationScheme"/>.
+        /// </summary>
+        /// <typeparam name="THandler">The <see cref="IAuthenticationHandler"/> responsible for the scheme.</typeparam>
+        /// <param name="name">The name of the scheme being added.</param>
+        /// <param name="displayName">The display name for the scheme.</param>
+        public void AddScheme<THandler>(string name, string displayName) where THandler : IAuthenticationHandler
```

Comment:
Sure it will, it's what end users could call to add handlers, its analogous to what hosting does: https://github.com/aspnet/IISIntegration/blob/dev/src/Microsoft.AspNetCore.Server.IISIntegration/IISMiddleware.cs#L53

except that's adding directly to the scheme provider rather than during startup/configure services.

Keep in mind, there's NOTHING on the builder, so the question I'm asking now is if we should nuke all of the `Action<AuthenticationSchemeBuilder>`...  It was more useful when we had a bag on the builder that could be configured.